doc/en/components/date-picker.md

```diff
@@ -0,0 +1,90 @@
+
+## Usage
+
+The $PlatformShort$ Date Picker editor component consists of three default parts - the text area, a calendar drop-down button, and a clear button. The text area will display the selected date, stored in the `Value` property of the editor.
+
+Clicking the calendar button will open the calendar drop-down of the Date Picker. The calendar will open to show the month of the currently selected `Value`, if there is one. If a `Value` is not specified, the calendar will show the month corresponding to the current date. The month at the top can be clicked to show a list of months, and the same applies for the year. There are also arrow buttons to the left and right of the month and year visualization that will increment and decrement the shown month chronologically. You can change the color of the calendar button by setting the `IconColor` property of the control.
+
+The clear button is directly to the left of the calendar button, visualized by an X. When clicked, it will clear the `Value` of the Date Picker. The clear button's visibility can be toggled on and off by setting the `ShowClearButton` property of the Date Picker.
+
+Users can also customize the Date Picker's font by using the various text properties of the control. They are: `TextColor`, `TextFontFamily`, `TextFontSize`, `TextFontStyle`, and `TextFontWeight`.
+
+## $PlatformShort$ Date Picker Overview Example
+
+<div class="sample-container loading" style="height: 500px">
+    <iframe id="category-chart-overview-iframe" src='{environment:dvDemosBaseUrl}/editors/date-picker-overview' width="100%" height="100%" seamless frameBorder="0" onload="onXPlatSampleIframeContentLoaded(this);" alt="$PlatformShort$ Date Picker Overview Example"></iframe>
+</div>
+
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn"   data-iframe-id="multi-column-combobox-overview-iframe" data-demos-base-url="{environment:dvDemosBaseUrl}">View on StackBlitz
+    </button>
+<sample-button src="editors/date-picker/overview"></sample-button>
+
+</div>
+
+<!-- React, WebComponents -->
+## Dependencies
+When installing the Date Picker component, the core and inputs packages must also be installed.
+
+<pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
+npm install --save {PackageCore}
+npm install --save {PackageInputs}
+</pre>
+<!-- end: React, WebComponents -->
+
+## Required Modules
+
+The Date Picker component requires the following modules to be registered:
+
+```razor
+DatePickerModule.Register(IgniteUIBlazor);
+```
+
+```ts
+import { IgrDatePickerModule } from 'igniteui-react-inputs';
+
+IgrDatePickerModule.register();
+```
+
+```ts
+import { IgcDatePickerModule } from 'igniteui-webcomponents-inputs';
+import { ModuleManager } from 'igniteui-webcomponents-core';
+
+ModuleManager.register(
+    IgcDatePickerModule
+);
+```
+
+<div class="divider--half"></div>
+
+## Code Snippet
+
```

Comment:
explain the code snippet with a description and add some more things to - for example setting specific properties with comments